include/zephyr/net/dns_resolve.h

```diff
@@ -774,6 +786,19 @@ static inline int dns_resolve_service(struct dns_resolve_context *ctx,
  */
 struct dns_resolve_context *dns_resolve_get_default(void);
 
+#if defined(CONFIG_DNS_RESOLVER_PACKET_FORWARDING)
+/**
+ * @brief Installs the packet forwarding callback to the DNS resolving context.
+ *
+ * @details When this callback is installed, a received message from DNS server
+ * will be passed to the application that has requested this callback.
```

Comment:
`the application that has requested this callback` sounds a bit odd, maybe simply `a received message from DNS server
 will be passed to the callback.`?

Reply:
Plus, again please document all paramters.